docs/modules/operation/pages/topology/asset-topology-provider/introduction.adoc

```diff
@@ -0,0 +1,53 @@
+
+[[asset-topology]]
+= Asset Topology Provider
+
+{page-component-title} lets users define arbitrarily complex layered topologies using http://graphml.graphdrawing.org/[GraphML].
+Refer to the GraphML section of the {page-component-title} developers guide for details of how {page-component-title} interprets GraphML.
```

Comment:
We could make this a cross reference:
```suggestion
Refer to xref:development:topology/graphml.adoc#graphml[GraphML] for details of how {page-component-title} interprets GraphML.
```
Note that you will have to add the "graphml" anchor to the graphml.adoc file for the link to work.